CanonicalLtd/maas-docs
# Entrypoint for talisker gevent workers

Fixes https://github.com/ubuntudesign/base-squad/issues/107

QA
--

``` bash
./run
```

See that it mentions "Booting worker" 3 times. Go to http://localhost:8203 - browse around, check the site works. Then:

``` bash
docker build --tag maas-docs --build-arg COMMIT_ID=`git rev-parse HEAD` .
docker run -ti -p 8099:80 maas-docs
```

See that it mentions "Booting worker" 3 times. Go to http://localhost:8099 - check the site works.

Also, check the `X-VCS-Revision` shows correctly:

``` bash
$ curl -Is localhost:8099 | grep VCS
X-VCS-Revision: {commit_hash}
```
